.github/workflows/ci.yml

```diff
@@ -68,6 +77,9 @@ jobs:
             --env "RAISE_ON_WARNINGS=true" \
             --env "DEPENDABOT_TEST_ACCESS_TOKEN=${{ secrets.GITHUB_TOKEN }}" \
             --env "SUITE_NAME=${{ matrix.suite.name }}" \
+            --env "GITHUB_SHA=${{ github.sha }}" \
+            --env "CI_NODE_TOTAL=${{ matrix.suite.ci_node_total }}" \
```

Comment:
I wonder if ` --env "CI_NODE_TOTAL=${{ matrix.suite.ci_node_total || 1 }}"` and similarly below will work, so that we don't need to change the whole matrix, just python, and as a result we don't manually need change all required jobs for CI to be green in UI configuration.

Reply:
We can give it a try, but first I just want to get the builds passing before I clean things up. Keep the suggestions coming though, appreciate it!

Reply:
I think we don't even need to pass a default value, we just don't use them in those scripts 🤷